src/settings.test.js

```diff
@@ -0,0 +1,136 @@
+import { expect } from 'chai';
+import _ from 'lodash';
+import settings from './settings';
+import utils from './utils';
+
+
+describe('/settings', () => {
+  describe('.set()', () => {
+    describe('path is a string', () => {
+      it('sets chosen parameters with current values', () => {
+        const testSettings = _.cloneDeep(settings);
+        testSettings.set('modes.VD.kSigma', 3);
+        expect(testSettings.now.modes.VD.kSigma).to.be.equal(3);
+      });
+    });
+
+    describe('path is not a string (is an object)', () => {
+      it('makes diff with settings.now and sets diff', () => {
+        const testSettings = _.cloneDeep(settings);
+
+        const testDiffs = _.cloneDeep(settings);
+        testDiffs.set('modes.VD.kSigma', 3);
+        testSettings.set(testDiffs.now);
+        expect(testSettings.now.modes.VD.kSigma).to.be.equal(3);
+      });
+    });
+  });
+
+  describe('.get()', () => {
+    it('gets current parameter', () => {
+      const testSettings = _.cloneDeep(settings);
+      testSettings.now.modes.VD.kSigma = 1;
+      expect(settings.get('modes.VD.kSigma')).to.be.equal(1);
+    });
+  });
+
+  describe('.reset()', () => {
+    it('resets all parameters in default', () => {
```

Comment:
Another behavior to verify in a separate statement is notifying about changes during reset.